pkg/processor/trigger/kafka/types.go

```diff
@@ -297,6 +301,12 @@ func NewConfiguration(id string,
 			Field:   &newConfiguration.maxWaitHandlerDuringRebalance,
 			Default: 5 * time.Second,
 		},
+		{
+			Name:    "wait explicit ack during rebalance timeout",
+			Value:   newConfiguration.WaitExplicitAckDuringRebalanceTimeout,
+			Field:   &newConfiguration.waitExplicitAckDuringRebalanceTimeout,
+			Default: 1 * time.Millisecond,
```

Comment:
Is this enough as a default value?

Reply:
@TomerShor it really depends on system, but during testing it was enough